### PR TITLE
Commented on code and fix issues

### PR DESCRIPTION
**Pull request by parneetbatra**
Commented on code and fix issues.

### CODE REVIEW

**Comment by parneetbatra** [comment]
@aatish-rajkarnikar @chandrika-katikam @ttawan I commented code and fix some issue. Please approve, Thanks.

**Review by aatish-rajkarnikar — APPROVED** [review]
Everything is good!

File: README.md [hunk 1]
```diff
@@ -1,15 +1,46 @@
 ## GagaCoders
 
 ## Contribution
-- PRs
+#### Pull requests
```
Comment (ttawan) [inline README.md:4]: Why did you added those links to README ?

**Review by ttawan — CHANGES_REQUESTED** [review]
Please remove all links from readme

File: README.md (hunk 1 above)
Comment (parneetbatra) [reply README.md]: Removed links

**Pull request merged**